app/javascript/packs/application.js

```diff
@@ -11,6 +11,7 @@ require("channels")
 require("bootstrap")
 
 require("../includes/vote.js")
+require("../includes/save.js")
```

Comment:
this file name "save" seems to be very generic, we need to know what we are saving..
Can we change the name to make it clear?

Reply:
Resolved it